src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -265,7 +270,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   }
 
   /**
-   * Takes two arrays of WDL Files and generates any necessary JES inputs from them.
+   * Takes two arrays of remote and local WDL File paths and generates the necessary JESInput's.
```

Comment:
Sidenote, I'm not sure why my commit was included - that's now part of the job_avoidance branch.